### PR TITLE
[Integration][Clickup] Adds Clickup integration documentation

### PR DESCRIPTION
**Pull request by DeeStarks**
# Description

This PR adds the documentation for the new Clickup integration [here](https://github.com/port-labs/ocean/pull/768)

## Added docs pages

Please also include the path for the added docs

- Clickup (`/build-your-software-catalog/sync-data-to-catalog/project-management/clickup/`)

### CODE REVIEW

File: docs/build-your-software-catalog/sync-data-to-catalog/project-management/clickup/clickup.md [hunk 1]
```diff
@@ -0,0 +1,1193 @@
...
+| `initialize_port_resources`        | Default true, When set to true the integration will create default blueprints and the port App config Mapping. Read more about [initializePortResources](https://ocean.getport.io/develop-an-integration/integration-configuration/#initializeportresources---initialize-port-resources) |         | ❌       |
+| `identifier`                       | The identifier of the integration that will be installed                                                                                                                                                                                                                                 |         | ❌       |
+| `version`                          | The version of the integration that will be installed                                                                                                                                                                                                                                    | latest  | ❌       | `   |
+
+<br/>
+
```
Comment (mk-armah) [inline docs/build-your-software-catalog/sync-data-to-catalog/project-management/clickup/clickup.md:187]: format properly

File: docs/build-your-software-catalog/sync-data-to-catalog/project-management/clickup/clickup.md [hunk 2]
```diff
@@ -0,0 +1,1193 @@
...
+:::info Additional parameters
+In the example above, two additional parameters are used:  
+`createMissingRelatedEntities` - used to enable the creation of missing related entities in Port. This is useful when you want to create an entity and its related entities in one call, or if you want to create an entity whose related entity does not exist yet.
+
+`deleteDependentEntities` - used to enable deletion of dependent Port entities. This is useful when you have two blueprints with a required relation, and the target entity in the relation should be deleted. In this scenario, the delete operation will fail if this parameter is set to `false`. If set to `true`, the source entity will be deleted as well.
+:::
+
+### Configuration structure
+
```
Comment (mk-armah) [inline docs/build-your-software-catalog/sync-data-to-catalog/project-management/clickup/clickup.md:416]: i thin we are missing `SUPPORTED RESOURCES (Kind)`  tip here

**Review by mk-armah — COMMENTED** [review]
go over my comments

File: docs/build-your-software-catalog/sync-data-to-catalog/project-management/clickup/clickup.md (hunk 1 above)
Comment (DeeStarks) [reply docs/build-your-software-catalog/sync-data-to-catalog/project-management/clickup/clickup.md]: done

**Pull request closed without merging**